### DISCUSSION TITLE
Build rainbowkit as a single file

### DISCUSSION THREAD

**Discussion by Linch1** [rainbow-me/rainbowkit#2246] [Q&A] [answered]
Hello, in my react project i'm using rainbowkit, at build time i can see that many files are placed in the dist folder, and many of this files are coming from rainbowkit, between svgs and translations, there is a way to have rainbowkit outputting a single file when built ? so having only one chunk instead of many?

I've tried vite/parcel/webpack for build and all of three have the same result.
thanks in advance :)

**Comment by DanielSinclair** [comment, marked as answer]
This is expected code splitting behavior that is used to optimize initial-load performance. When using RainbowKit in a project like Next, the bundler should bundle the imported files to prevent network roundtrips. Typically you'd want to leverage this code splitting, but you can likely change your bundle settings to work around this.
